postgresql/resource_postgresql_role.go

```diff
@@ -286,6 +293,15 @@ func resourcePostgreSQLRoleCreate(db *DBConnection, d *schema.ResourceData) erro
 		}
 	}
 
+	areLogsEnabled := d.Get(roleEnableLogsAttr).(bool)
+	if areLogsEnabled {
+		sql := "SET log_statement TO 'none'; SET log_min_duration_statement TO -1; SET log_min_error_statement TO 'log'; SET pg_stat_statements.track_utility = 'off';"
+
+		if _, err := txn.Exec(sql); err != nil {
+			return fmt.Errorf("could not disable logs for %s: %w", roleName, err)
```

Comment:
I feel like you inverse the boolean no?

if `areLogsEnabled` is set to true, you disable the logs